README.md

```diff
@@ -515,6 +515,7 @@ Follows best practices and conventions to provide you a SOLID development experi
 * [Humanizer](https://github.com/Humanizr/Humanizer) - Humanizer meets all your .NET needs for manipulating and displaying strings, enums, dates, times, timespans, numbers and quantities.
 * [Humidifier](https://github.com/jakejscott/Humidifier) - Write and maintain AWS Cloudformation templates using C#.
 * [JqueryDataTablesServerSide](https://github.com/fingers10/JqueryDataTablesServerSide) - ASP.NET Core Server Side Processing library for Jquery DataTables with Multiple Column Filtering, Sorting and Pagination at database level with Excel Export and TagHelper support.
+* [Lexical.FileSystem](https://github.com/tagcode/Lexical.FileSystem) - Virtual IFileSystem interfaces, and implementations. Adaptable with IFileProvider.
```

Comment:
Might also be worth mentioning System.IO.Abstractions?

Reply:
One entry per PR.

Reply:
Perhaps you should create a new document # Section for filesystems and fileproviders.